blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -696,3 +697,35 @@ def test_coalesce(sqla):
         compute(coalesce(t.A, 'z'), {t: sqla}, return_type=list) ==
         [('a',), ('z',), ('c',)]
     )
+
+
+@pytest.mark.xfail(reason="Having clause not yet implemented")
+def test_having(big_sql):
+    '''
+    The blaze expression generated in 'res' below is incorrect. It invokes the
+    where clause but it should be using a HAVING clause. Need to add HAVING
+    clause to blaze
+
+        print(res)
+        SELECT tbl0."A", sum(tbl0."B") AS quant
+            FROM tbl0
+        WHERE sum(tbl0."B") > 125
+        GROUP BY
+            tbl0."A"
+
+    The correct expression uses HAVING for aggregate function:
+
+        SELECT tbl0."A", sum(tbl0."B") AS quant
+            FROM tbl0
+        GROUP BY
+            tbl0."A" HAVING sum(tbl0."B") > 125
+    '''
+    ds = discover(big_sql)
+    nn = symbol('nn', ds)
+    g1 = by(nn['A'], quant=nn.B.sum())
+    g1_res = odo(compute(g1, big_sql), list)
```

Comment:
Can you remove the `odo` call and convert this to `compute(g1, big_sql, return_type=list)` instead, for consistency?